src/script/filterByRating.js

```diff
@@ -0,0 +1,20 @@
+export class FilterByRating {
+
+    filter (challenge) {
+        const minRating = document.querySelectorAll(".rating.min-rating>.rating-star.active").length;
+        const maxRating = document.querySelectorAll(".rating.max-rating>.rating-star.active").length;
+
+        if (challenge.rating >= minRating && challenge.rating <= maxRating){
+            
+            return true;
+        
+        } else {
+        
+            return false;
+        }
+    }
+
+    checkDOM () {
```

Comment:
You can make this function check if the min and maxRating is within the "allowed" range.
minRating > 0, maxRating <= 5;
return true; regardless is kinda unnecessary.

Reply:
True, it's an easy fix!